R/aaa_models.R

```diff
@@ -922,11 +957,13 @@ get_pred_type <- function(model, type) {
   check_model_exists(model)
   pred_name <- paste0(model, "_predict")
   if (!any(pred_name != rlang::env_names(get_model_env()))) {
-    rlang::abort(glue::glue("`{model}` does not have any `pred` methods in parsnip."))
+    cli::cli_abort(
+      "{.arg {model}} does not have any {.fn pred} methods in parsnip."
```

Comment:
```suggestion
      "{.arg {model}} does not have any {.arg pred} methods in parsnip."
```

Or maybe `.fn predict`? I don't believe `pred()` is a function.